src/client/client.cpp

```diff
@@ -782,11 +782,20 @@ void Client::initLocalMapSaving(const Address &address,
 		return;
 	}
 
-	const std::string world_path = porting::path_user
-		+ DIR_DELIM + "worlds"
-		+ DIR_DELIM + "server_"
+	std::string world_path;
+#define set_world_path(hostname) \
+	world_path = porting::path_user \
```

Comment:
FYI: This would be more like C++:
```C++
auto set_world_path = [&] (const std::string &hostname) {
	world_path = porting::path_user +
		DIR_DELIM + "worlds" +
		DIR_DELIM + "server_" +
		hostname + "_" + std::to_string(address.getPort());
};
```

Reply:
Ah, I am new to C++ from C, thanks for that.